tests/hpc/pdsh_slave.pm

```diff
@@ -36,6 +36,7 @@ sub run ($self) {
     $self->switch_user('nobody');
     my $genders_plugin = get_var('PDSH_GENDER_TEST') ? '-g type=genders-test' : '';
     assert_script_run("pdsh -R mrsh $genders_plugin -w $server_hostname ls / &> /tmp/pdsh.log");
+    assert_script_run("test -s /tmp/pdsh.log");
```

Comment:
ah OK, I did not think of using it this way for checking the output file.
Would it make sense to do the same for the MPI tests? Some time back those produced a file with 'Hello world' printed from an MPI basic application to a  file, so perhaps this test -s trick would be good there too?

Reply:
sure. i review other modules as well and i will do so in small chucks. To answer more specific i would have to check how the behavior is MPI again. Also this approach is very minimalistic. We might think a better solution later on

Reply:
yeah definitely; specifically for the MPI jobs parsing of the output would be definitely better idea. On my TODO list for the MPI there was an idea of running way more sophisticated/real-life MPI application and there the parsing of the output would be imo crucial.